Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio-jolt"
-version = "0.1.1"
+version = "0.2.0"
```

Comment:
Is there a backward compatibility issue from prev version?  if not can bump to `0.1.2`

Reply:
Couldn't be sure since it is very complicated with all expression parsing and execution but probably it is backwards compatible

Reply:
Sounds good. let's bump to `0.2.0`